source/setup.py

```diff
@@ -228,6 +228,7 @@ def getRecursiveDataFiles(dest,source,excludes=()):
 		("libArm64/%s"%version, glob("libArm64/*.dll") + glob("libArm64/*.exe")),
 		("waves", glob("waves/*.wav")),
 		("images", glob("images/*.ico")),
+		("fonts", glob("fonts/*.ttf")),

```

Comment:
Should we include the files with otf extension here, too? If not, I wonder why we do allow them in the source but we don't include them as per the setup

Reply:
We don't have any `otf` files currently. The code to load them will at least handle them we needed them in the future.